content/en/docs/Tasks/update-operator-to-next-version.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Update operator to next version"
+date: 2021-03-09
+weight: 2
+description: >
+   Given an operator at version n, what should I do to create a version n+1.
+--- 
+
```

Comment:
IHMO we need to;
- explain how to do by as a how to' using the operator bundle format\
- we need to explain the manual steps but also we can describe how sdk cli tool can help the users. I would indeed first explain via sdk and then manually.